pkg/expapi/types.go

```diff
@@ -134,3 +136,40 @@ type HorizontalPodAutoscalerList struct {
 
 	Items []HorizontalPodAutoscaler `json:"items"`
 }
+
+type DerivedNodeMetrics struct {
+	api.TypeMeta   `json:",inline"`
+	api.ObjectMeta `json:"metadata,omitempty"`
+
+	NodeMetrics      MetricsWindows            `json:"nodemetrics"`
+	SystemContainers []DerivedContainerMetrics `json:"systemcontainers"`
+}
+
+type DerivedContainerMetrics struct {
+	Name    string         `json:"name"`
```

Comment:
What is name supposed to be?  I would recommend being explicit because of the potential confusion between metric name and container name.  `containerName`